src/Site.js

```diff
@@ -6,12 +6,12 @@ const path = require('path');
 const Promise = require('bluebird');
 const ProgressBar = require('progress');
 const walkSync = require('walk-sync');
-const VariablePreprocessor = require('./lib/markbind/src/preprocessors/variablePreprocessor');
-const njUtil = require('./lib/markbind/src/utils/nunjuckUtils');
-const injectHtmlParser2SpecialTags = require('./lib/markbind/src/patches/htmlparser2');
-const injectMarkdownItSpecialTags = require(
-  './lib/markbind/src/lib/markdown-it/markdown-it-escape-special-tags');
-const utils = require('./lib/markbind/src/utils');
+
+const injectHtmlParser2SpecialTags = require('markbind/src/patches/htmlparser2');
```

Comment:
Can remove the import on Line 1.

Reply:
not too sure what you meant here,

`const cheerio = require('cheerio'); require('./lib/markbind/src/patches/htmlparser2');`?
`cheerio` is used in one instance below, the patch is necessary to ensure unit tests load with the patch

Reply:
changed `require('./lib/markbind/src/patches/htmlparser2');` to `require('markbind/src/patches/htmlparser2');` though if that's what you meant

Reply:
We're doing the same import on this line 🙊 

But if it's clearer that we're doing the patch, then I'm ok with the duplicate import.

Reply:
> But if it's clearer that we're doing the patch

🤦‍♂️ 🤦‍♂️ I'll leave it in then!